### PR TITLE
Chore/modularize typedefs and resolvers

### PR DESCRIPTION
**Pull request by digorath**
typedefs for different models. are now defined in their own respective files

### CODE REVIEW

File: src/typeDefs/constructionSite.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    category: String
+    "Grund der Baustelle"
+    cause: String
+    "Beschreibung der Baustelle"
+    description: String
+    "Richtung der Baustelle"
+    direction: String
+    "(Voraussichtliches) Ende der Baustelle"
+    endDate: String
+    "URL einer zugöhrigen Bilddatei"
```
Comment (donni106) [inline src/typeDefs/constructionSite.ts:16]: zugöhrigen -> zugehörigen

File: src/typeDefs/typeDefs.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+  type Mutation {
+    createRoadwork(name: String!, description: String): Roadwork!
+    deleteRoadwork(id: ID!): Roadwork!
+    updateRoadwork(id: ID!, name: String, description: String): Roadwork!
+  }
+`;
+
+export const typeDefs = [
+  _baseGQL, // query and mutation need to be first, because their types are extended by others
```
Comment (donni106) [inline src/typeDefs/typeDefs.ts:21]: doesn't the underscore indicate something unused, which isn't the case here?

**Review by donni106 — APPROVED** [review]

File: src/typeDefs/typeDefs.ts (hunk 2 above)
Comment (digorath) [reply src/typeDefs/typeDefs.ts:21]: i have come across multiple conventions using the underscore prefix for various reasons.

i only know it to mean "unused variable" if the name consists of exclusively underscores.

in this case i just wanted to make this one variable stand out among the rest and be correctly sorted to the beginning as well.

if you have a better suggestion feel free to leave it here.

File: src/typeDefs/constructionSite.ts (hunk 1 above)
Comment (digorath) [reply src/typeDefs/constructionSite.ts]: fixed

File: src/typeDefs/typeDefs.ts (hunk 2 above)
Comment (donni106) [reply src/typeDefs/typeDefs.ts:21]: good to know, well reasoned

**Pull request merged**